common/src/main/java/com/github/dominik48n/party/command/AcceptCommand.java

```diff
@@ -20,11 +20,18 @@
 import com.github.dominik48n.party.api.Party;
 import com.github.dominik48n.party.api.PartyAPI;
 import com.github.dominik48n.party.api.player.PartyPlayer;
+import com.github.dominik48n.party.config.PartyConfig;
 import java.util.Optional;
 import org.jetbrains.annotations.NotNull;
 
 public class AcceptCommand extends PartyCommand {
 
+    private final @NotNull PartyConfig config;
+
+    public AcceptCommand(final @NotNull PartyConfig config) {
```

Comment:
Lombok Konstruktor für mehr Übersichtlichkeit (persönliche Präferenz)

Reply:
The reason why this is not implemented can be found in the comment in the pull request. 

But I remove the public constructor :P